tests/charts/airflow_core/test_worker.py

```diff
@@ -373,6 +373,17 @@ def test_should_create_default_affinity(self):
             docs[0],
         )
 
+    def test_runtime_class_name_values_are_configurable(self):
+        docs = render_chart(
+            values={
+                "workers": {"runtimeClassName": "nvidia"},
+            },
+            show_only=["templates/workers/worker-deployment.yaml"],
+        )
+
+        runtimeclassname = jmespath.search("spec.template.spec.runtimeClassName", docs[0])
+        assert runtimeclassname == "nvidia"
```

Comment:
You can just do an `assert jmespath.search("spec.template.spec.runtimeClassName", docs[0]) == "nvidia"` here